homeassistant/components/devolo_home_control/binary_sensor.py

```diff
@@ -128,6 +129,7 @@ def __init__(
 
         self._key = key
         self._attr_is_on = False
+        self._attr_name = f"Button {key}"
```

Comment:
Hmm this was previously not prefixed. What does this have to do with the new naming style?